x/metadata/keeper/keeper.go

```diff
@@ -8,107 +8,21 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
 	"github.com/cosmos/gogoproto/proto"
 
 	"github.com/provenance-io/provenance/x/metadata/types"
 )
 
-// MetadataKeeperI is the internal state api for the metadata module.
```

Comment:
why remove this interface section?

Reply:
We weren't using it anywhere. Also, it was located in the `keeper` package, so if something else wanted to use it, they'd have to import that package which we usually want to avoid because it's a common way to end up with circular references. If anything wanted to do stuff with the metadata keeper, they'd want to define their own interface for use in their own stuff; that's basically the `expected_keepers.go` pattern.

And by having it, it's just another place that needs updating if we ever want to add/remove/change existing keeper methods.